Documentation/Contributors/TestingGuide/README.md

```diff
@@ -51,71 +44,87 @@ All new code should have 100% code coverage and should pass all tests. Always ru
 
-![](7.jpg)
+#### Run All Tests with WebGL Stub
 
-This uses a query parameter to select the test/suite to run so refreshing the page will run just that test/suite again.
+This makes all WebGL calls a noop and ignores test expectations that rely on reading back from WebGL. This allows running the tests on CI where a reasonable WebGL implementation is not available and still getting full code coverage albeit not all verification.
 
-Often when developing, it is useful to run only one suite to save time, instead of all the tests, and then run all the tests before opening a pull request.
+`npm run test-webgl-stub`
 
-### Run Only WebGL Tests
+#### Run Only WebGL Tests
 
 Suites can have a category associated with them. This option runs all tests in the `WebGL` category, which includes all tests that use WebGL (basically anything that requires creating a `Viewer`, `CesiumWidget`, `Scene`, or `Context`).
 
-### Run Only Non-WebGL Tests
+`npm run test-webgl`
+
+#### Run Only Non-WebGL Tests
 
 Likewise, this option runs all tests not in the WebGL category.
 
 Perhaps surprisingly, this is the bulk of CesiumJS tests, which include math and geometry tests, imagery provider tests, data source tests, etc.
 
 These tests run quickly (for example, 15 seconds compared to 60) and are very reliable across systems since they do not rely on the underlying WebGL implementation, which can vary based on the browser, OS, driver, and GPU.
 
-### Run All Tests against Combined File (Run All Tests against Combined File with Debug Code Removed)
+`npm run test-non-webgl`
+
+#### Run All Tests Against the Minified Release Version of CesiumJS
 
 Most test options load CesiumJS using the individual source files in the `Source` directory, which is great for debugging.
 
-However, many users build apps using the built Cesium.js in `Build/Cesium` (which is created, for example, by running `npm run combine`). This option runs the tests using this instead of individual CesiumJS source files.
+However, many users build apps using the built Cesium.js in `Build/Cesium` (which is created, for example, by running `npm run minifyRelease`). This option runs the tests using this instead of individual CesiumJS source files. The release version has `DeveloperError` exceptions optimized out so this test option makes `toThrowDeveloperError` always pass. See the [Build Guide](https://github.com/CesiumGS/cesium/blob/main/Documentation/Contributors/BuildGuide/README.md#build-scripts) for all the CesiumJS build options.
 
-The **Run All Tests against Combined File with Debug Code Removed** is the same except it is for use with the release version of the built Cesium.js (which is created, for example, by running `npm run combineRelease`). The release version has `DeveloperError` exceptions optimized out so this test option makes `toThrowDeveloperError` always pass.
+`npm run test-release`
+
+When testing against built Cesium.js, the specs need to be built as well with `npm run build-specs`.
```

Comment:
I think this line should go before `npm run test-release`, since that's the order they should be run in.